benchmark/bench-eltwise-mult-mod.cpp

```diff
@@ -174,5 +177,48 @@ BENCHMARK(BM_EltwiseMultModAVX512IFMAInt)
 
 //=================================================================
 
+#ifdef HEXL_HAS_AVX512IFMA
+
+// state[0] is the degree
+// state[1] is the input_mod_factor
+static void BM_EltwiseMultModMontAVX512IFMAIntEConv(
+    benchmark::State& state) {  //  NOLINT
+
+  size_t input_size = state.range(0);
+  size_t input_mod_factor = state.range(1);
+  uint64_t modulus = (1ULL << 50) + 7;  // 1125899906842631
+  auto op1 = GenerateInsecureUniformRandomValues(input_size, 0,
+                                                 input_mod_factor * modulus);
+  auto op2 = GenerateInsecureUniformRandomValues(input_size, 0,
+                                                 input_mod_factor * modulus);
+  AlignedVector64<uint64_t> output(input_size, 3);
+
+  int r = 51;  // R = 2251799813685248
+  // mod(2251799813685248*2251799813685248;1125899906842631)
```

Comment:
I'm personally not so fond of the magic numbers in the comments. Perhaps
```
uint64_t R_square_mod_q = MultiplyMod(1ULL << r, 1ULL << r, modulus);
```
would work?

Reply:
It doesn't work because inputs are bigger than modulus in MultiplyMod:
  HEXL_CHECK(x < modulus, "x " << x << " >= modulus " << modulus);
  HEXL_CHECK(y < modulus, "y " << y << " >= modulus " << modulus);